src/PKSim.CLI.Core/MinimalImplementations/CLIDialogCreator.cs

```diff
@@ -78,7 +78,9 @@ public string AskForFolder(string title, string directoryKey, string defaultDire
          throw new NotSupportedException();
       }
 
-      public string AskForInput(string caption, string text, string defaultValue = null, IEnumerable<string> forbiddenValues = null, IEnumerable<string> predefinedValues = null, string iconName = null)
+      public string AskForInput(string caption, string text, string defaultValue = null, IEnumerable<string> forbiddenValues = null,
```

Comment:
I feel it's time for you to rebase on develop before doing anything

Reply:
@msevestre  Hmmm.. I have just rebased on develop, but something seems to have gone very wrong: the diff is actually gone up to 124 files....

Reply:
@msevestre ...but if I try to make a PR between the branch and develop (instead of the branch and the feature branch), the diif seems to be exactly correct (10 files). I am not sure why this is happening. 

The issue with this would be that since the feature branch in Core has not been merged to develop, it would cause a problem with the nuget packages...